packages/react/toolbar/src/Toolbar.tsx

```diff
@@ -177,6 +176,7 @@ const ToolbarToggleGroup = React.forwardRef<
     return (
       <ToggleGroupPrimitive.Root
         data-orientation={context.orientation}
+        dir={context.dir}
```

Comment:
We need to pass this down to `ToggleGroupPrimitive` as we now consistent provide a fallback `dir` value. Without doing so `ToolbarToggleGroup` renders `dir="ltr"` even when `<Toolbar.Root dir="rtl"/>`, thus we pass it explicitly via context to prevent the fallback behaviour.